controllers/ramenconfig.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 The RamenDR authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	ramendrv1alpha1 "github.com/ramendr/ramen/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var (
+// cachedRamenConfig ramendrv1alpha1.RamenConfig
```

Comment:
(nit) minor cleanups, please deal with the same in the subsequent patches on topic.

Reply:
Yes, done.